app.py

```diff
@@ -1,8 +1,14 @@
-from fastapi import FastAPI
+from fastapi import FastAPI, HTTPException
 from fastapi.responses import JSONResponse
 from cfenv import AppEnv
+from operations import perform_operation
+from pydantic import BaseModel
 
 
+class OperationRequest(BaseModel):
```

Comment:
You need to create these pydantic models in different module

Reply:
ok sure